docs/Interoperable-RA-TLS-cert-edvidence-formats.md

```diff
@@ -4,17 +4,19 @@ Interoperable RA-TLS X.509 Cert and Evidence Formats
 # X.509 Cert with Evidence and Endorsements Extensions
 
 A X.509 cert for RA-TLS can either be self-signed or CA-signed. It must contain an extension for evidence, and can (optionally) contain an extension for endorsements. The evidence and endorsement extensions are defined in the latest [TCG DICE specification draft](https://members.trustedcomputinggroup.org/wg/DICE/document/36858).
+
 - Evidence (required): X.509 cert extension, a byte string of encoded tagged CBOR data
-    - OID: `tcg-dice-tagged-evidence` (2.23.133.5.4.9)
+    - OID: `tcg-dice-conceptual-message-wrapper` (2.23.133.5.4.9)
     - CBOR tag: a CBOR tag registered with IANA or some other registry serves as evidence format ID
-    - CBOR data: this data covered by the tag contains the evidence (including custom claims). Its format is defined by the CBOR tag. 
+    - CBOR data: this data covered by the tag contains the evidence (including custom claims). Its format is defined by the CBOR tag.
+    - Note: the DICE specification states that the criticality flag of this extension SHOULD be marked critical, but popular TLS libraries (such as openssl and mbedtls) will fail in verification of an X.509 cert with this extension if its criticality flag is set. To overcome this limitation, the criticality flag will be cleared.
 - Endorsement (optional): X.509 cert extension, also a byte string of encoded tagged CBOR data. When this extension is present, its value can be used as endorsements for verification of the evidence in the evidence extension.
     - OID: `tcg-dice-endorsement-manifest` (2.23.133.5.4.2)
     - CBOR tag: the same tag as that for evidence extension
-    - CBOR data: this data covered by the tag contains the endorsements for verification of the evidence in the same cert. Its format is defined by the CBOR tag. 
+    - CBOR data: this data covered by the tag contains the endorsements for verification of the evidence in the same cert. Its format is defined by the CBOR tag.
 
 # Evidence Custom Claims
-The evidence extension must include a `pubkey-hash` claim, and can optionally include a `nonce` claim:
+The evidence extension must include a `pubkey-hash` claim, and can optionally include a `nonce` claim. All other custom claims will be ignored. How the custom claims are stored in the evidence is dependent on the specific evidence format definition.
```

Comment:
> `How the custom claims are stored in the evidence is dependent on the specific evidence format definition.`

Do we want this sentence? It doesn't really describe anything, and it only confused me on the first read.

Reply:
> > `How the custom claims are stored in the evidence is dependent on the specific evidence format definition.`
> 
> Do we want this sentence? It doesn't really describe anything, and it only confused me on the first read.

Thanks! I'll remove it.